### PR TITLE
iisadministration PowerShell reference update

### PR DESCRIPTION
**Pull request by TimShererWithAquent**
[1870688](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1870688)

This is generated changes only.

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2022-ps/iisadministration/Get-IISSiteBinding.md [hunk 1]
```diff
@@ -92,14 +93,17 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/iisadministration/Get-IISSiteBinding.md:97]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/iisadministration/New-IISSiteBinding.md [hunk 2]
```diff
@@ -194,13 +194,18 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/iisadministration/New-IISSiteBinding.md:198]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/iisadministration/Remove-IISSiteBinding.md [hunk 3]
```diff
@@ -133,10 +133,17 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (dariomws) [inline docset/winserver2022-ps/iisadministration/Remove-IISSiteBinding.md:137]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2022-ps/iisadministration/Set-IISCentralCertProviderCredential.md [hunk 4]
```diff
@@ -75,7 +75,7 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## OUTPUTS
 
-### None
+### System.Object
```
Comment (dariomws) [inline docset/winserver2022-ps/iisadministration/Set-IISCentralCertProviderCredential.md:78]: This cmdlet has a string parameter as input.

**Review by dariomws — CHANGES_REQUESTED** [review]
Minor suggestions.

I also think that these updates should apply to other versions of Windows (2016 and 2019).

**Comment by TimShererWithAquent** [comment]
@dariomws - Thanks for your help. As for previous versions, the plan is to move the changes into the previous versions once we get the current version up to date -

**Comment by TimShererWithAquent** [comment]
@tiburd - I've made the changes and it looks like we have 3 approvals. Let me know if I need to do anything else.

**Review by dariomws — APPROVED** [review]

**Comment by dariomws** [comment]
Thanks @TimShererWithAquent!

**Pull request merged**